.docker/Dockerfile-alpine

```diff
@@ -1,20 +1,21 @@
 FROM alpine:3.20
 
-RUN addgroup -S ory; \
-    adduser -S ory -G ory -D -H -s /bin/nologin && \
-    apk upgrade --no-cache && \
+RUN <<HEREDOC
+    apk upgrade --no-cache
```

Comment:
Do you really need this?

```suggestion
```

Likewise the next apk command runs `--upgrade` which AFAIK isn't needed.

Reply:
done